modules/AWS/eks/outputs.tf

```diff
@@ -38,3 +38,9 @@ output "max_cluster_size" {
 output "min_cluster_size" {
   value = module.eks.node_groups["appNodes"]["scaling_config"][0]["min_size"]
 }
+
+output "availability_zone" {
+  # TODO we are assuming the used availability zone is always aws-region"a"
```

Comment:
Do we need to refactor now, or if not how much this assumption is valid?

Reply:
This assumption is valid. It is not a straight refactor. I had a different solution in place but it wasn't testable with `terratest`. I think it is good for now and this PR.